README.md

```diff
@@ -78,114 +78,97 @@ For guidance on developing for PowerToys, please read the [developer docs](/doc/
+Notably, based on the community feedback received, PowerToys has re-introduced the highly-requested ability to activate Shortcut Guide via holding the `Win` key. PowerToys also now allows various commands in PowerToys Run to be used in either the universal English phrasing or system-localized translation. The great feedback the community provides is invaluable in helping PowerToys continually grow and improve as a product.
 
-An experimental version of [PowerToys (v0.46)][github-prerelease-link] was released the week of September 13th. Download and install it to try out the latest improvements to our Video Conference Mute utility! All updates from the v0.45.0 release will still apply in v0.46.0.
+An experimental version of PowerToys (v0.48) will be released the week of October 4th, introducing improvements to our Video Conference Mute utility! All updates from the v0.47.0 release will still apply in v0.48.
 
-#### Highlights from v0.45
+#### Highlights from v0.47
 
 **General**
 
-- Settings and OOBE windows updated with Fluent UX! We hope you enjoy the new modern feel of the application menus as we align our product with upcoming Windows 11 interfaces. Special thanks to @niels9001 for driving the development of this UI, along with many thanks to the various community members who offered constant feedback and adjustments to make this a truly spectacular update!
-- Added button to settings which displays version history. Thanks @niels9001!
-- Signed PowerToysSetupCustomActions.dll.
-- Improved auto-update experience. Thanks @niels9001!
-- Aligned OOBE theme color with Settings theme color. Thanks @niels9001!
-- Adjusted labeling of "Restart as Administrator" button to "Restart PowerToys as Administrator" to avoid ambiguity in meaning. Thanks @niels9001!
-- Added colored icons to settings sidebar. Thanks @niels9001!
-- Fixed accessibility issue in OOBE where Microsoft Docs and PowerToys release notes links could not be navigated to via keyboard. Thanks @niels9001!
-- Fixed settings header alignment. Thanks @niels9001!
-- Fixed text under updates section to be visible when in light mode. Thanks @niels9001!
-- Updated "Learn More" text to be more descriptive. Thanks @niels9001!
-- Updated "Read more" text on updates to be more descriptive. Thanks @niels9001!
-- Added link to documentation in system tray. Thanks @BenConstable9!
-- Fixed error caused by file in use issues when installing PowerToys.
-- Fixed issue where opening settings from start menu didn't work when PowerToys was run as admin. Thanks @davidegiacometti!
+- Fixed issue with new updates changing the PowerToys install location.
+- Fixed settings with NumberBox elements overlapping the delete button.
+- Fixed issue with the bug report tool not generating .zip files.
+- Fixed issue with disabled shortcuts looking the same as when enabled. Thanks @niels9001!
```

Comment:
This was self-inflicted - we do however have a new shortcut experience in Settings, I'd mention that here instead of this line